### PR TITLE
Add Documentation for CS:S Assets

### PR DESCRIPTION
**Pull request by Arkanoid0**
Automatic css asset downloading scripts documented

### CODE REVIEW

**Comment by osinmv** [comment]
I will take care of this guide.

**Comment by Gocnak** [comment]
A more recent script by spx_ does VPK re-packing (making each game extract to their own VPKs). It would be beneficial to update this script to also use it:
```sh
#!/bin/sh

app_id=232330
name=css
vpks="$PWD/tmp/cstrike/cstrike_pak_dir.vpk"
to_extract="models/|materials/|sound/"
bindir="$PWD/tmp/bin"
vpk_command="env LD_LIBRARY_PATH=$bindir $bindir/vpk_linux32"
momentum_custom_dir=momentum/custom

./steamcmd.sh +login anonymous +force_install_dir tmp +app_update $app_id +exit
mkdir -p extracted
cd extracted
for vpk in $vpks; do
	echo extracting $vpk...
	filelist=$(eval $vpk_command l "$vpk" | grep -E '^('"$to_extract"')' | tr '\n' ',')
	echo $filelist | xargs -d ',' dirname | xargs -d '\n' mkdir -p
	echo $filelist | eval xargs -d ',' $vpk_command x "$vpk"
done
cd ..
eval $vpk_command extracted
mkdir -p "$momentum_custom_dir/$name"
mv extracted.vpk "$momentum_custom_dir/$name"
rm -rf extracted tmp
```

**Comment by Arkanoid0** [comment]
> A more recent script by spx_ does VPK re-packing (making each game extract to their own VPKs). It would be beneficial to update this script to also use it:
> 
> ```shell
> #!/bin/sh
> 
> app_id=232330
> name=css
> vpks="$PWD/tmp/cstrike/cstrike_pak_dir.vpk"
> to_extract="models/|materials/|sound/"
> bindir="$PWD/tmp/bin"
> vpk_command="env LD_LIBRARY_PATH=$bindir $bindir/vpk_linux32"
> momentum_custom_dir=momentum/custom
> 
> ./steamcmd.sh +login anonymous +force_install_dir tmp +app_update $app_id +exit
> mkdir -p extracted
> cd extracted
> for vpk in $vpks; do
> 	echo extracting $vpk...
> 	filelist=$(eval $vpk_command l "$vpk" | grep -E '^('"$to_extract"')' | tr '\n' ',')
> 	echo $filelist | xargs -d ',' dirname | xargs -d '\n' mkdir -p
> 	echo $filelist | eval xargs -d ',' $vpk_command x "$vpk"
> done
> cd ..
> eval $vpk_command extracted
> mkdir -p "$momentum_custom_dir/$name"
> mv extracted.vpk "$momentum_custom_dir/$name"
> rm -rf extracted tmp
> ```

This is great, but its written in bash, it won't run under windows, so its not going to be very useful for the majority of the player base.

**Comment by osinmv** [comment]
Arkanoid0 do you want to finish it then? I honestly thought that I need just to write a guide on how to use your thing.

**Comment by Arkanoid0** [comment]
> Arkanoid0 do you want to finish it then? I honestly thought that I need just to write a guide on how to use your thing.

the reason i haven't is i was planning to completely remake the tool to be more user friendly and have a UI and such. really goc should have opened a issue on my repo with that code, but its w/e

File: _posts/2019-08-23-guide_install-css-assets.md [hunk 1]
```diff
@@ -7,4 +8,8 @@ tags:
   - cs:s
 ---
 
-TODO: Show how to install CS:S assets from SteamCMD
+# Using the Auto Downloader
+Download and run the automatic script found [here](https://github.com/Arkanoid0/Momentum-Mod-SteamCMD-Utilities). Copy the css folder to custom/ within your momentum install.
```
[the remark this answers is not in the document]
Comment (CoopJax) [reply _posts/2019-08-23-guide_install-css-assets.md:12]: Yeah, it looks ok... Just put it in more detail.

**Review by CoopJax — COMMENTED** [review]
This looks ok, just

**Comment by braem** [comment]
Any updates on this?

**Comment by Gocnak** [comment]
This was done and now has a proper guide, closing unless there's further developments.

**Pull request closed without merging**